src/modules/dashboard/components/CoinMachine/BuyTokens/BuyTokens.tsx

```diff
@@ -201,7 +201,7 @@ const BuyTokens = ({
   const getFormattedCost = useCallback(
     (amount) => {
       const decimalCost = new Decimal(amount)
-        .times(salePriceData.coinMachineCurrentPeriodPrice)
+        .times(salePriceData?.coinMachineCurrentPeriodPrice || '0')
```

Comment:
Shouldn't this actually be _OR `1`_, because if you multiply by `0`, you'll always get `0`, but if you multiply by `1` you'll just get the amount.

Depends on what you want the fallback to be.

Reply:
I think the default value should be `0`. If it's `0` it will give an indication that something is off more that if it defaults to `1`.... But I am happy to hear a counter-argument